modules/nextflow/src/main/groovy/nextflow/processor/PublishDir.groovy

```diff
@@ -222,7 +222,7 @@ class PublishDir {
     protected void apply0(Set<Path> files) {
         assert path
 
-        final retryOpts = session.config.navigate('nextflow.publish.retryPolicy') as Map ?: Collections.emptyMap()
+        final retryOpts = session.config.navigate('workflow.output.retryPolicy') as Map ?: Collections.emptyMap()
```

Comment:
What's if there's an usage of this setting using `nextflow.publish.retryPolicy`?

Reply:
We can keep the old syntax if you want, but I don't think we need to. I don't think anyone has been using these settings

Reply:
the problem is that `nextflow.publish.retryPolicy` than it would be better to keep it as it as even if I agree may not be used.

Reply:
something like this?

```groovy
        final retryOpts = session.config.navigate('workflow.output.retryPolicy') as Map ?: session.config.navigate('workflow.publish.retryPolicy') as Map ?: Collections.emptyMap()
```

Reply:
I guess the problem is that the component is not aware if it's related to `output` or `publish` scope. What about passing the type (it could be just a string) when creating that instance ?

Reply:
ok, agreed. let's use `workflow.output`. let's mention in the docs `workflow.publish.retryPolicy` has been replaced by `workflow.output.retryPolicy`

Reply:
and reporting a warning if `workflow.publish.retryPolicy` is used in place of `workflow.output.retryPolicy`

Reply:
done